src/scrcmd_c.c

```diff
@@ -101,6 +101,7 @@
 #include "constants/trainers.h"
 #include "render_window.h"
 #include "overlay_01_021F1AFC.h"
+#include "frontier/frontier.h"
```

Comment:
double frontier/frontier.h include

Reply:
This is fine as-is

Reply:
wdym? it adds a header which has already been explicitly included here